Assets/Plugins/README.md

```diff
@@ -1,105 +1,101 @@
+* [Unity v2020.3.48f1 or later](https://unity.com/download)
 
-### Mobile
+Desktop requirements:
 
-- Register your application with Sky Mavis to get `YOUR_APP_ID` and `YOUR_DEEP_LINK_SCHEMA`
-- Request permission to use Mavis ID
-- Go to Developer Console > your app > App Permission > Mavis ID > Request Access
+* A game distributed through [Mavis Hub](https://hub.skymavis.com).
+* [.Net v3.1](https://dotnet.microsoft.com/en-us/download/dotnet/3.1) or later.
 
-[Head to the detail guide](https://docs.skymavis.com/comming-soon) to acquired `YOUR_APP_ID` and `YOUR_DEEP_LINK_SCHEMA`
+Mobile requirements:
 
-`Android` : Settings in Unity as described in the [Customize platform-specific settings](https://docs.skymavis.com/mavis/mavis-id/guides/integrate-unity#customize-platform-specific-settings) section in the integration guide.
+* An app created in the [Developer Console](https://developers.skymavis.com/console/applications/).
+* Permission to use Mavis ID. Request in **Developer Console > your app > App Permission > Sky Mavis Account (OAuth 2.0) > Request Access**.
+* A client ID that you can find in **Developer Console > Products > ID Service > Client ID**.
+* A redirect URI registered in **Developer Console > Products > ID Service > Redirect URI**.
 
-### Initialization
+* To deploy to Android, [Android API level 24](https://developer.android.com/about/versions/nougat) or later.
+* To deploy to iOS, [iOS 12.0](https://developer.apple.com/ios/) or later.
 
-- Initialize SDK for desktop and mobile
+## Installation
+
+1. Download the latest `ID.unitypackage` release from this repository.
+2. Import the file by selecting the Unity menu option **Assets > Import package > Custom Package** and importing `ID.unitypackage`.
+3. In Unity Editor, go to **Build Settings**, then choose the platform you are deploying, then click **Switch Platform**.
+4. Configure [platform-specific settings](https://docs.skymavis.com/mavis/mavis-id/guides/unity-sdk#step-2-install-the-sdk).
+
+## Initialization
+
+### Mobile
 
 ```csharp
-    public async void OnInitialized()
-    {
-#if UNITY_STANDALONE
-        string sessionId = GetArg("-sessionId");
-        string idPort = GetArg("-hubPort");
-        if (!string.IsNullOrEmpty(idPort) && int.TryParse(idPort, out var port))
-        {
-            SM.MavisId.Init(sessionId, port);
-            float startSec = Time.realtimeSinceStartup;
-            while (!SM.MavisId.IsConnected && Time.realtimeSinceStartup - startSec <= 15) await Task.Yield();
-            if (SM.MavisId.IsConnected)
-            {
-                Debug.Log("Connected to MH Overlay");
-            }
-            else
-            {
-                Debug.LogError("Can't connect to MH Overlay");
-            }
-        }
-#else
-        SM.MavisId.Init(AppId, DeeplinkSchema);
-#endif
-    }
+void Start()
+{
+    // Client ID registered in the Mavis ID settings in the Developer Console
+    string appId = "cbabcb00-9c99-404b-a6e4-c76b3b59f0d8";
```

Comment:
I don't know that does Sec team allow us public client id although it's just a test project

Reply:
We can hide it in env variables, what would the syntax be like?

Reply:
// Client ID registered with Mavis ID
string appID = ${YOUR_CLIENT_ID}

This is the popular that I use